### PR TITLE
Onboarding Portal

### PR DESCRIPTION
**Pull request by golmek**
Proposed changes in doc to include references to Onboarding Portal

### CODE REVIEW

File: src/pages/docs/creators/javascript/guides/switching-to-mainnet.md [hunk 1]
```diff
@@ -75,6 +75,24 @@ It may be useful to export your Golem wallet address to an external application
...
 
+#### Acquiring MATIC/ETH and GLM tokens directly via Golem Onboarding Portal
+
+1. Visit our [Onboarding Portal](https://glm.golem.network/)
+2. Click **Get Started** or **Get GLM** button
+3. Follow the instructions provided by the Portal
+
+#### Acquiring MATIC/ETH and GLM tokens from YAGNA context via Golem Onboarding Portal
+
+1a. For Polygon network, enter following command into your Terminal window (Yagna has to be running in the background):
```
Comment (jalas167) [inline src/pages/docs/creators/javascript/guides/switching-to-mainnet.md:86]:
```suggestion
1. For Polygon network, enter following command into your Terminal window (Yagna has to be running in the background):
```

Comment (jalas167) [inline src/pages/docs/creators/javascript/guides/switching-to-mainnet.md:86]:
```suggestion
1. For Polygon network, enter following command into your Terminal window (Yagna has to be running in the background):
```

Comment (jalas167) [inline src/pages/docs/creators/javascript/guides/switching-to-mainnet.md:86]:
```suggestion
1. For Polygon network, enter following command into your Terminal window (Yagna has to be running in the background):
```

File: src/pages/docs/creators/javascript/guides/switching-to-mainnet.md [hunk 2]
```diff
@@ -75,6 +75,24 @@ It may be useful to export your Golem wallet address to an external application
...
+```bash
+yagna payment fund --network=polygon
+```
+1b. Alternatively for Ethereum network, enter following command:
```
Comment (jalas167) [inline src/pages/docs/creators/javascript/guides/switching-to-mainnet.md:90]:
```suggestion
Alternatively for the Ethereum network, enter the following command:
```

File: src/pages/docs/creators/javascript/guides/switching-to-mainnet.md (hunk 1 above)
Comment (jalas167) [inline src/pages/docs/creators/javascript/guides/switching-to-mainnet.md:86]:
```suggestion
1. For the Polygon network, enter the following command into your Terminal window (Yagna has to be running in the background):
```

File: src/pages/docs/creators/javascript/guides/switching-to-mainnet.md [hunk 3]
```diff
@@ -75,6 +75,24 @@ It may be useful to export your Golem wallet address to an external application
 
 {% /alert %}
 
+#### Acquiring MATIC/ETH and GLM tokens directly via Golem Onboarding Portal
```
Comment (jalas167) [inline src/pages/docs/creators/javascript/guides/switching-to-mainnet.md:78]:
```suggestion
#### Acquiring MATIC/ETH and GLM tokens directly via the Golem Onboarding Portal
```

File: src/pages/docs/creators/javascript/guides/switching-to-mainnet.md [hunk 4]
```diff
@@ -75,6 +75,24 @@ It may be useful to export your Golem wallet address to an external application
...
+
+1. Visit our [Onboarding Portal](https://glm.golem.network/)
+2. Click **Get Started** or **Get GLM** button
+3. Follow the instructions provided by the Portal
+
+#### Acquiring MATIC/ETH and GLM tokens from YAGNA context via Golem Onboarding Portal
```
Comment (jalas167) [inline src/pages/docs/creators/javascript/guides/switching-to-mainnet.md:84]:
```suggestion
#### Acquiring MATIC/ETH and GLM tokens from the YAGNA context via the Golem Onboarding Portal
```

File: src/pages/docs/creators/javascript/guides/switching-to-mainnet.md [hunk 5]
```diff
@@ -75,6 +75,24 @@ It may be useful to export your Golem wallet address to an external application
...
+
+1a. For Polygon network, enter following command into your Terminal window (Yagna has to be running in the background):
+```bash
+yagna payment fund --network=polygon
+```
+1b. Alternatively for Ethereum network, enter following command:
+```bash
+yagna payment fund --network=mainnet
+```
+2. Follow instructions presented into newly opened Web Page of your default browser
```
Comment (jalas167) [inline src/pages/docs/creators/javascript/guides/switching-to-mainnet.md:94]:
```suggestion
2. Follow the instructions presented on the newly opened Web Page of your default browser
```

File: src/pages/docs/creators/python/examples/tools/managing-golem-wallet.md [hunk 6]
```diff
@@ -104,6 +104,30 @@ yagna payment fund
...
+{% tab label="Ethereum mainnet" %}
+
+```bash
+yagna payment fund --network=mainnet
+```
+
+{% /tab %}
+{% /tabs %}
+
+This commands should open your Web Browser with Golem Onboarding Portal web page. Follow the instructions presented there.
```
Comment (jalas167) [inline src/pages/docs/creators/python/examples/tools/managing-golem-wallet.md:129]:
```suggestion
This command should open your Web Browser with the Onboarding Portal web page. Follow the instructions presented there.
```

File: src/pages/docs/creators/python/examples/tools/managing-golem-wallet.md [hunk 7]
```diff
@@ -104,6 +104,30 @@ yagna payment fund
 Golem will transfer test tokens from our custom faucet (a service that transfers test tokens to an address that asks for them).
 Note, you also need to enable your accounts' sender mode, which is done using `yagna payment init`.
 
+## Obtaining funds to your mainnet account
+
+You can top up your wallet with MATIC/ETH and GLMs by running following command:
```
Comment (jalas167) [inline src/pages/docs/creators/python/examples/tools/managing-golem-wallet.md:109]:
```suggestion
You can top up your wallet with MATIC/ETH and GLMs by running the following command:
```

File: src/pages/docs/creators/python/guides/switching-to-mainnet.md [hunk 8]
```diff
@@ -63,14 +63,32 @@ You can top up your wallet with the following command:
 yagna payment fund
 ```
 
-### On mainnet
+### On paid network
```
Comment (jalas167) [inline src/pages/docs/creators/python/guides/switching-to-mainnet.md:66]:
```suggestion
### On paid networks
```

File: src/pages/docs/creators/python/guides/switching-to-mainnet.md [hunk 9]
```diff
@@ -63,14 +63,32 @@ You can top up your wallet with the following command:
...
 
 {% alert level="info" %}
 
 It may be useful to export your Golem wallet address to an external application that enables transactions on the internet. You can find instructions on how to back up and restore your wallet and export wallet address [here](/docs/creators/python/examples/tools/golem-wallet-backup).
 
 {% /alert %}
 
+#### Acquiring MATIC/ETH and GLM tokens directly via Golem Onboarding Portal
```
Comment (jalas167) [inline src/pages/docs/creators/python/guides/switching-to-mainnet.md:74]:
```suggestion
#### Acquiring MATIC/ETH and GLM tokens directly via the Golem Onboarding Portal
```

File: src/pages/docs/creators/python/guides/switching-to-mainnet.md [hunk 10]
```diff
@@ -63,14 +63,32 @@ You can top up your wallet with the following command:
...
+
+1. Visit our [Onboarding Portal](https://glm.golem.network/)
+2. Click **Get Started** or **Get GLM** button
+3. Follow the instructions provided by the Portal
+
+#### Acquiring MATIC/ETH and GLM tokens from YAGNA context via Golem Onboarding Portal
```
Comment (jalas167) [inline src/pages/docs/creators/python/guides/switching-to-mainnet.md:80]:
```suggestion
#### Acquiring MATIC/ETH and GLM tokens from the YAGNA context via the Golem Onboarding Portal
```

File: src/pages/docs/creators/python/guides/switching-to-mainnet.md [hunk 11]
```diff
@@ -63,14 +63,32 @@ You can top up your wallet with the following command:
...
+
+1a. For Polygon network, enter following command into your Terminal window (Yagna has to be running in the background):
+```bash
+yagna payment fund --network=polygon
+```
+1b. Alternatively for Ethereum network, enter following command:
+```bash
+yagna payment fund --network=mainnet
+```
+2. Follow instructions presented into newly opened Web Page of your default browser
```
Comment (jalas167) [inline src/pages/docs/creators/python/guides/switching-to-mainnet.md:90]:
```suggestion
2. Follow the instructions presented on the newly opened Web Page of your default browser
```

File: src/pages/docs/creators/python/guides/switching-to-mainnet.md [hunk 12]
```diff
@@ -63,14 +63,32 @@ You can top up your wallet with the following command:
...
+2. Click **Get Started** or **Get GLM** button
+3. Follow the instructions provided by the Portal
+
+#### Acquiring MATIC/ETH and GLM tokens from YAGNA context via Golem Onboarding Portal
+
+1a. For Polygon network, enter following command into your Terminal window (Yagna has to be running in the background):
+```bash
+yagna payment fund --network=polygon
+```
+1b. Alternatively for Ethereum network, enter following command:
```
Comment (jalas167) [inline src/pages/docs/creators/python/guides/switching-to-mainnet.md:86]:
```suggestion
Alternatively for the Ethereum network, enter the following command:
```

File: src/pages/docs/creators/python/guides/switching-to-mainnet.md [hunk 13]
```diff
@@ -63,14 +63,32 @@ You can top up your wallet with the following command:
...
 
+#### Acquiring MATIC/ETH and GLM tokens directly via Golem Onboarding Portal
+
+1. Visit our [Onboarding Portal](https://glm.golem.network/)
+2. Click **Get Started** or **Get GLM** button
+3. Follow the instructions provided by the Portal
+
+#### Acquiring MATIC/ETH and GLM tokens from YAGNA context via Golem Onboarding Portal
+
+1a. For Polygon network, enter following command into your Terminal window (Yagna has to be running in the background):
```
Comment (jalas167) [inline src/pages/docs/creators/python/guides/switching-to-mainnet.md:82]:
```suggestion
1. For the Polygon network, enter the following command into your Terminal window (Yagna has to be running in the background):
```

**Review by jalas167 — COMMENTED** [review]
recommendations added

File: src/pages/docs/creators/javascript/guides/switching-to-mainnet.md (hunk 2 above)
Comment (golmek) [reply src/pages/docs/creators/javascript/guides/switching-to-mainnet.md:90]: OK

File: src/pages/docs/creators/javascript/guides/switching-to-mainnet.md (hunk 1 above)
Comment (golmek) [reply src/pages/docs/creators/javascript/guides/switching-to-mainnet.md:86]: OK

File: src/pages/docs/creators/javascript/guides/switching-to-mainnet.md (hunk 3 above)
Comment (golmek) [reply src/pages/docs/creators/javascript/guides/switching-to-mainnet.md:78]: OK

File: src/pages/docs/creators/javascript/guides/switching-to-mainnet.md (hunk 4 above)
Comment (golmek) [reply src/pages/docs/creators/javascript/guides/switching-to-mainnet.md:84]: OK

File: src/pages/docs/creators/javascript/guides/switching-to-mainnet.md (hunk 5 above)
Comment (golmek) [reply src/pages/docs/creators/javascript/guides/switching-to-mainnet.md:94]: OK

File: src/pages/docs/creators/python/examples/tools/managing-golem-wallet.md (hunk 6 above)
Comment (golmek) [reply src/pages/docs/creators/python/examples/tools/managing-golem-wallet.md:129]: OK

File: src/pages/docs/creators/python/examples/tools/managing-golem-wallet.md (hunk 7 above)
Comment (golmek) [reply src/pages/docs/creators/python/examples/tools/managing-golem-wallet.md:109]: OK

File: src/pages/docs/creators/python/guides/switching-to-mainnet.md (hunk 8 above)
Comment (golmek) [reply src/pages/docs/creators/python/guides/switching-to-mainnet.md:66]: OK

File: src/pages/docs/creators/python/guides/switching-to-mainnet.md (hunk 9 above)
Comment (golmek) [reply src/pages/docs/creators/python/guides/switching-to-mainnet.md:74]: OK

File: src/pages/docs/creators/python/guides/switching-to-mainnet.md (hunk 10 above)
Comment (golmek) [reply src/pages/docs/creators/python/guides/switching-to-mainnet.md:80]: OK

File: src/pages/docs/creators/python/guides/switching-to-mainnet.md (hunk 11 above)
Comment (golmek) [reply src/pages/docs/creators/python/guides/switching-to-mainnet.md:90]: OK

File: src/pages/docs/creators/python/guides/switching-to-mainnet.md (hunk 12 above)
Comment (golmek) [reply src/pages/docs/creators/python/guides/switching-to-mainnet.md:86]: OK

File: src/pages/docs/creators/python/guides/switching-to-mainnet.md (hunk 13 above)
Comment (golmek) [reply src/pages/docs/creators/python/guides/switching-to-mainnet.md:82]: OK

**Review by jalas167 — APPROVED** [review]

**Pull request merged**